src/librustc_mir/interpret/memory.rs

```diff
@@ -56,12 +57,12 @@ pub struct Memory<'a, 'mir, 'tcx: 'a + 'mir, M: Machine<'a, 'mir, 'tcx>> {
     /// the wrong type), so we let the machine override this type.
     /// Either way, if the machine allows writing to a static, doing so will
     /// create a copy of the static allocation here.
-    alloc_map: M::MemoryMap,
+    crate alloc_map: M::MemoryMap,
```

Comment:
Why is this needed?

Reply:
changed to `pub(super)`, interning needs it to remove allocations from `Memory` when moving them to the global memory.